docs/source/chaincode_lifecycle.md

```diff
@@ -149,11 +149,14 @@ consistent across organizations:
 - **Collection Configuration:** The path to a private data collection definition
   file associated with your chaincode. For more information about private data
   collections, see the [Private Data architecture reference](https://hyperledger-fabric.readthedocs.io/en/{BRANCH}/private-data-arch.html).
-- **Initialization:** All chaincode need to contain an ``Init`` function that is
-  used to initialize the chaincode. By default, this function is never executed.
-  However, you can use the chaincode definition to request that the ``Init``
-  function be callable. If execution of ``Init`` is requested, fabric will ensure
-  that ``Init`` is invoked before any other function and is only invoked once.
+- **Initialization:** If you use the low level APIs provided by the Fabric Chaincode
+  Shim API, your chaincode needs to contain an ``Init`` function that is used to
+  initialize the chaincode. By default, this function is never executed. However,
+  you can use the chaincode definition to request that the ``Init`` function be
+  callable. If execution of ``Init`` is requested, fabric will ensure that
```

Comment:
Similar comments as above.

It doesn't indicate that Init is 'callable'. It indicates if Init is required to be called by a user.

Reply:
This is still an open comment. You may be able to simply copy/paste the Init text that we improved above. Or describe it well once, and reference the description in the other location.

Reply:
I agree with this. The chaincode for developers topic is supposed to be a getting started topic. Three in depth paragraphs about the Init function are out of place, and are more about the CC lifecycle than getting started with development. As a result, I will move the bulk of the material to this topic and link out.